src/index.js

```diff
@@ -0,0 +1,43 @@
+import Store from '../modules/store';
+import Ui from '../modules/ui';
+import './style.css';
+
+class Task {
+  constructor(description, index) {
+    this.Description = description;
+    this.completed = false;
+    this.index = index;
```

Comment:
- [ ]please ensure that  Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique
![Screenshot (258)_LI](https://user-images.githubusercontent.com/65298048/179926712-4d0a22f4-3d50-42a1-a699-ffe4f3555876.jpg)